README.md

```diff
@@ -75,11 +75,7 @@ IDE Options | Description
 Create VSCode Project | As well as the CMake files, also create the appropriate Visual Studio Code project files.
 Debugger | Use the specified debugger in the IDE
 
+## Development
 
-
-
-
-
-
-
-
+* use a virtual environment to ensure dependencies are isolated from your system's
+* additional dependencies: `pip install pytest`
```

Comment:
I think this might need rewording, as it currently reads as though `pip install pytest` is the command you run to "use a virtual environment..." ?

Reply:
Oh, I hadn't realised that these are actually separate bullet points, and not simply a single sentence line-wrapped over two lines!